k8s/celery-beat.yaml.j2

```
@@ -0,0 +1,40 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ CELERY_SCHEDULER_NAME }}
+spec:
+  replicas: {{ CELERY_SCHEDULER_REPLICAS }}
+  selector:
+    matchLabels:
+      app: {{ CELERY_SCHEDULER_NAME }}
+  template:
+    metadata:
+      labels:
+        app: {{ CELERY_SCHEDULER_NAME }}
+    spec:
+      volumes:
+      - name: developer-portal-fs
+        persistentVolumeClaim:
+          claimName: {{ APP_PVC_NAME }}
+      containers:
+        - name: {{ CELERY_SCHEDULER_NAME }}
+          image: "{{ APP_IMAGE }}:{{ APP_IMAGE_TAG }}"
+          imagePullPolicy: {{ APP_IMAGE_PULL_POLICY }}
+          volumeMounts:
+            - name: developer-portal-fs
+              mountPath: {{ APP_MOUNT_PATH }}
+          command:
+            - "celery"
+          args:
+            - "--app=developerportal.apps.staticbuild"
+            - worker
```

```suggestion
            - beat
```
This needs to run as `beat` rather than another `worker` node

---

Fixed in a4c7d02f6ec2119197cd5267ddd3f23f91264cf3